LayoutTests/security/contentSecurityPolicy/plugins-types-allows-youtube-plugin-replacement.html

```diff
@@ -1,27 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Hm... don't we want to keep YouTube plugin replacement? I think that's still supported, right?

Reply:
All of these tests are skipped as far as I can tell. Maybe @achristensen07 can share some insight.

Reply:
YouTube plugin replacement is still supported by all browsers and we should keep and maybe even be working towards re-enabling the tests.

Reply:
I kept the four YouTube plugin replacement tests. I will see about enabling them in a separate PR. I also addressed a minor issue caught by CI where a test depended on one of the removed resources. Hopefully it's all good now.